ci/requirements-dev-3.6.yml

```diff
@@ -23,3 +23,4 @@ dependencies:
     - hdfs>=2.0.0
     - clickhouse-driver
     - clickhouse-sqlalchemy
+    - git+https://github.com/googledatalab/pydatalab@v1.1
```

Comment:
I'm curious why you are using the Datalab libraries instead of google-cloud-python?

Reply:
@tswast , its what I was exposed to in some internal demos of the tooling we can use to access gcp.

Is google-cloud-python the right thing to use?

Reply:
Yeah, it's what we recommend at https://cloud.google.com/bigquery/docs/reference/libraries#client-libraries-install-python Datalab is great, but it's not really built to be used outside of the context of their notebook environment.